py/dml/dmlc.py

```diff
@@ -490,6 +499,8 @@ def show_illegal_attributes(option, opt, value, parser):
               % (options.split_c_file,))
         sys.exit(1)
 
+    dml.globals.coverity = bool(os.getenv('DMLC_COVERITY')) or options.coverity
```

Comment:
Keep both the flag and the environment variable, or have only one of them? If the latter, which one?